browser/DaisyPayments.js

```diff
@@ -13,14 +20,26 @@ export default class DaisyPayments extends ClientPayments {
     this.override = override;
   }
 
-  with(withGlobals) {
-    return new DaisyPayments({
-      manager: this.manager,
-      override: this.override,
-      withGlobals: { ...this.withGlobals, ...withGlobals },
-    });
-  }
-
+  /**
+   * If this class is instantiated only with {@link module:common~PaymentGroup#identifier}
+   * this call is necessary to fetch the payment group's manager data.
+   * @async
+   * @returns {this} - Return self instance.
+   *
+   * @example
+   *
+   * const daisy = new DaisySDK.Payments({
+   *   manager: { identifier: ... }, withGlobals: { web3 },
+   * });
+   * await daisy.sync() // required
+   *
+   * @example
+   *
+   * const daisy = await DaisySDK.initPayments({
+   *   manager: { identifier: ... }, withGlobals: { web3 },
+   * }); // sync not required here.
+   *
```

Comment:
I don't exactly get what this example is showing here. If we use `new DaisySDK.Payments()` we have to `sync()` and if we use `DaisySDK.initPayments()` we don't? The explanation makes it sounds like it depends on the absence or presence of some additional arguments